ajax/lib/snoocore/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "snoocore",
+  "version": "3.2.0",
+  "filename": "Snoocore-browser.min.js",
+  "description": "A minimal and complete JavaScript driver for the Reddit API.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/trevorsenior/snoocore.git"
+  },
+  "author": "Trevor Senior",
+  "homepage": "https://snoocore.readme.io/",
+  "keywords": [
+    "reddit",
+    "api",
+    "snoocore",
+    "wrapper",
+    "client"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/trevorsenior/snoocore.git",
+    "basePath": "dist",
+    "files": [
+      "Snoocore-browser*"
```

Comment:
Please use `"**/"` instead of `"Snoocore-browser*"` so that we can also grab other files in future, we use this pattern for the dist folder, thanks.